x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/__jest__/processors/append.test.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { act } from 'react-dom/test-utils';
+import { setup, SetupResult, getProcessorValue } from './processor.helpers';
+
+// Default parameter values automatically added to the URI parts processor when saved
+const defaultAppendParameters = {
```

Comment:
I’m seeing some inconsistencies with the values defined in the default parameters object among the tests. For example, here, we define `if` and `tag` as `undefined`, but in the `bytes` test we do not. It is not surfacing as a test failure since `toEqual` recursively checks the equality of all fields, and not for object identity. Maybe we can have a default set of "common" default parameters that we can reuse in the tests (e.g., for `if`, `tag`, `description`, and `ignore_failure`)?

Reply:
Done.